### PR TITLE
Update README in preparation for the upcoming 5.0.0 release

### PR DESCRIPTION
**Pull request by olivierdagenais**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +25,14 @@ Allows you to use a TFVC repository as an SCM in Jenkins jobs. At the moment, th
 
 The plugin will automatically create a workspace in TFS/Team Services and map a work folder (in the Jenkins workspace) to it.
 
+### Git
+The TFS plug-in for Jenkins enhances the Git plug-in for Jenkins by adding some integration features:
+* A push trigger, to request builds of specific commits in Git repositories without needing to schedule SCM polling
+* A build step that adds a "build pending" status to the associated pull request and/or commit in TFS/Team Services
+* A post-build action that add a "build completed" status to the associated pull request and/or commit in TFS/Team Services
+* Some endpoints for TFS/Team Services to use to activate the integration.
```
Comment (yacaovsnc) [inline README.md]: As a user, do I need to know what are those "some endpoints" at all?  If yes, I'd appreciate a link to learn more about those endpoints.  If not really, this creates some confusion in my mind already...

**Comment by DavidStaheli** [comment]
Looks good!  I will add the part about the Git Refspec later.

File: README.md (hunk 1 above)
Comment (olivierdagenais) [inline README.md]: > As a user, do I need to know what are those "some endpoints" at all? If yes, I'd appreciate a link to learn more about those endpoints.

Would 3045fcd satisfy your curiosity?

File: README.md [hunk 2]
```diff
@@ -69,13 +80,41 @@ Versions 3.2.0 and earlier of the plugin required a command line tool to be inst
...
+        4. You can use the _Description_ field to record details about the PAT, such as its intended collection, the selected authorization scopes and expiration date.  For example: `fabrikam-fiber-inc, code read+status, expires 2017-08-05`
+        5. Click **OK**   
+2. Add the collection URL and associate it with the right credential:
+    1. Select **Jenkins** > **Manage Jenkins** > **Configure System**
+    2. Scroll to **TFS/Team Services** and click **Add**
+        1. If using Team Services, the value of the _Collection URL_ field should omit `/DefaultCollection`.
+        2. Select the associated `Credentials` value created earlier.
+        3. Click **Test Connection**.
+    3. Click **Save**
+
```
Comment (yacaovsnc) [inline README.md]: Hmm, I followed this section to completion.  Very clear instruction!

Then as I continued down this ReadMe file, I realized the next section is about TFVC support.  Am I done configuring Jenkins to integrate with VSTS/TFS?  I'd appreciate some help to actually define this Jenkins job.  If it's not appropriate for a README, maybe a link to another doc?

Comment (olivierdagenais) [inline README.md]: > Am I done configuring Jenkins to integrate with VSTS/TFS? I'd appreciate some help to actually define this Jenkins job.

I was in the middle of writing 8e1bd5c when you posted that comment: you're too fast! :smile:

**Comment by yacaovsnc** [comment]
LGTM!!!  Great job, really nice to see a PR gated build on Jenkins!

**Pull request merged**